speakeasy/windows/winemu.py

```diff
@@ -98,7 +98,8 @@ def __init__(self, config, logger=None, exit_event=None, debug=False):
         self.wintypes = windef
         # OS resource managers
         self.regman = RegistryManager(self.get_registry_config())
-        self.fileman = FileManager(self.get_filesystem_config())
+        self.fileman = FileManager(self.get_filesystem_config(),
+                self.config_modules, self.command_line, self)
```

Comment:
```suggestion
        self.fileman = FileManager(self.get_filesystem_config(), self.config_modules, self.command_line, self)
```

i dk what the max line length is in this project, but its probably more than 80 chars